source/styles/typography/index.md

```diff
@@ -1,188 +1,184 @@
-  <p>
-    The following snippet of text is <em>rendered as italicized text</em>.
-  </p>
-  
-  <p>
-    The following snippet of text is <a href="#">rendered as link text</a>.
-  </p></section> 
-  
-  <hr />
-  
-  <b>List Samples</b> <section> 
-  
-  <div class="row">
-    <div class="col-sm-6 col-md-3">
-      <h4>
-        Unordered
-      </h4>
-      
-      <ul>
-        <li>
-          Lorem ipsum
-        </li>
-        <li>
-          Consectetur <ul>
-            <li>
-              Phasellus
-            </li>
-            <li>
-              Purus
-            </li>
-          </ul>
-        </li>
-        
-        <li>
-          Faucibus
-        </li>
-        <li>
-          Aenean sit
-        </li>
-      </ul>
-    </div>
-    
-    <!-- /col -->
-    
-    <div class="col-sm-6 col-md-3">
-      <h4>
-        Ordered
-      </h4>
-      
-      <ol>
-        <li>
-          Lorem
-        </li>
-        <li>
-          Consectetur
-        </li>
-        <li>
-          Aenean sit
-        </li>
-        <li>
-          Eget
-        </li>
-      </ol>
-    </div>
-    
-    <!-- /col -->
-    
-    <div class="clearfix visible-sm">
-    </div>
-    
-    <div class="col-sm-6 col-md-3">
-      <h4>
-        Unstyled
-      </h4>
-      
-      <ul class="list-unstyled">
-        <li>
-          Lorem
-        </li>
-        <li>
-          Consectetur
-        </li>
-        <li>
-          Nulla <ul>
-            <li>
-              Phasellus
-            </li>
-            <li>
-              Purus
-            </li>
-          </ul>
-        </li>
-        
-        <li>
-          Faucibus
-        </li>
-        <li>
-          Aenean
-        </li>
-        <li>
-          Eget
-        </li>
-      </ul>
-    </div>
-    
-    <!-- /col -->
-    
-    <div class="col-sm-6 col-md-3">
-      <h4>
-        Inline
-      </h4>
-      
-      <ul class="list-inline">
-        <li>
-          Lorem ipsum
-        </li>
-        <li>
-          Phasellus
-        </li>
-        <li>
-          Nulla
-        </li>
-      </ul>
-    </div>
-    
-    <!-- /col -->
+# Typography
+
+PatternFly uses Open Sans, an open-licensed, sans serif, typeface designed by Steve Matteson. The type family includes a complete 897 character set, along with Cyrillic character sets. There are ten type-face variants available with this font. Use the CSS rule “@font-face” to include these fonts in your application styles. For more information, check out [Mozilla Developer Network's @font-face Summary](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face).
+
+![#open-sans-image](img/open-sans-image.png)
+___
+
+## Typographic Styles
+
+The below style samples represent commonly-used typographic layouts. Check out more samples including `<dd>/<dl>` description lists, alignment classes, unstyled lists, and inline lists, here: [PatternFly Typography test page](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/typography-2.html).
```

Comment:
Instead of using "here:", recommending: 

Check out more samples including description lists, alignment classes, unstyled lists, and inline lists, on the PatternFly Typography test page.

Reply:
fixed.